steamos-waydroid-installer.sh

```diff
@@ -246,7 +249,7 @@ else
 	echo -e "$current_password\n" | sudo mkdir /var/lib/waydroid &> /dev/null
 	echo -e "$current_password\n" | sudo -S ln -s ~/waydroid/images /var/lib/waydroid/images &> /dev/null
 	echo -e "$current_password\n" | sudo -S ln -s ~/waydroid/cache_http /var/lib/waydroid/cache_http &> /dev/null
-
+    echo -e "$current_password\n" | sudo -S ln -s ~/waydroid/overlay /var/lib/waydroid/overlay &> /dev/null
```

Comment:
```suggestion
```

I don't think this is needed either?

also this line has space indentation (should be tabs)